doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -4,16 +4,20 @@
 You can set up an automatic cleanup job in {CENTRAL}.
 
 .Procedure
-. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. In {CENTRAL}, go to *Manage* > *Jobs*.
 . Click *New Job*.
-. Enter a name, due date, time, and the following command into the *Type* text field.
+. Enter a *Business Key*, *Due On*, *Retries*, and the following command into the *Type* field.
 +
 [source]
 ----
 org.jbpm.executor.commands.LogCleanupCommand
 ----
 
-. Click *Add Parameter* if you want to use the <<logcleanupcommand_parameters_{context}, parameters>>, enter a parameter in the key section and enter a parameter value in the value section.
+. If you want to use the <<logcleanupcommand_parameters_{context}, parameters>>:
++
+.. Open the *Advanced* tab.
+.. Click *Add Parameter*.
+.. Enter a parameter in the *Key* column and enter a parameter value in the *Value* column.
 . Click *Create* to finalize the job creation wizard.
```

Comment:
Very minor, but for minimalism you may want to just say "Click *Create*. Totally up to you :)

Reply:
Done